inc/vmem.h

```diff
@@ -1,37 +1,40 @@
 #ifndef VMEM_H
 #define VMEM_H
 
-#include <iostream>
+#include <cstdint>
 #include <deque>
 #include <map>
 
 #define VMEM_RAND_FACTOR 91827349653
 // reserve 1MB of space
 #define VMEM_RESERVE_CAPACITY 1048576
 
+#define PTE_BYTES 8
+
+//Virtual Address: 57 bit (9+9+9+9+9+12), rest MSB bits will be used to generate a unique VA per CPU.
+//PTL5->PTL4->PTL3->PTL2->PTL1->PFN
+
+constexpr uint64_t MINOR_FAULT_PENALTY = 200;
```

Comment:
This seems like another knob to add to the configuration file later on.

Reply:
I chose not to add it because I considered it an estimate used for testing purposes, but it just takes 4 additional lines to make it configurable. Should I do that now, or leave it until later?

Reply:
That's valid. Does this value change in any other PRs to be based on the memory system latencies or is it staying static?

I mainly left this to remind myself to add it post-review, but you can add it now if it doesn't take too much effort.

Reply:
This is a static penalty. If we want to make it more dynamic or more realistic, we would need to actually build out some more proper support.

Reply:
I went ahead and made it configurable. I figure, why not?

Reply:
Thanks! Yeah, I don't see how it could hurt.